File: epoxy-processortest/src/test/resources/ViewProcessorTest/annotationsAreCopied_java/SourceView.java
package com.airbnb.epoxy;

import android.content.Context;
import android.widget.FrameLayout;

@ModelView(defaultLayout = 1)
public class SourceView extends FrameLayout {

  public SourceView(Context context) {
    super(context);
  }

  @SuppressWarnings("unused")
  @ModelProp String sectionId;
}

File: epoxy-processortest/src/test/resources/ViewProcessorTest/inheritingAttributesWorksCorrectlyJavaSources/SourceView.kt
package com.airbnb.epoxy

import android.content.Context
import android.view.View

@ModelView(
    autoLayout = ModelView.Size.MATCH_WIDTH_WRAP_HEIGHT,
    baseModelClass = AirEpoxyModel::class
)
class SourceView(context: Context) : BaseView(context) {

    var sectionId: String? = null
        @ModelProp set
}

@ModelView(autoLayout = ModelView.Size.MATCH_WIDTH_WRAP_HEIGHT)
open class BaseView(context: Context) : View(context) {

    @ModelProp
    fun baseViewProp(prop: Int) {
    }
}

